From: Simon McVittie <smcv@debian.org>
Date: Sat, 4 Nov 2017 14:50:49 +0000
Subject: build: Stop using GNOME_COMPILE_WARNINGS

There is no point in checking for compiler warnings when we don't
compile any C or C++ code.

Signed-off-by: Simon McVittie <smcv@debian.org>
Bug-Debian: https://bugs.debian.org/829972
Forwarded: https://bugzilla.gnome.org/show_bug.cgi?id=789906
---
 configure.ac | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 1db8499..0f762db 100644
--- a/configure.ac
+++ b/configure.ac
@@ -12,8 +12,6 @@ GLIB_GSETTINGS
 
 PKG_PROG_PKG_CONFIG
 
-GNOME_COMPILE_WARNINGS([maximum])
-
 YELP_HELP_INIT
 
 AM_PATH_PYTHON([2.7])
